lte/gateway/python/integ_tests/s1aptests/test_attach_service_with_multi_pdns_and_bearers.py

```diff
@@ -0,0 +1,180 @@
+"""
```

Comment:
can you add a verification that the flow rules are preserved after switching back to connected state?